test/pylib/manager_client.py

```diff
@@ -104,70 +118,91 @@ async def replicas(self) -> int:
         resp = await self.client.get_text("/cluster/replicas")
         return int(resp)
 
-    async def running_servers(self) -> List[str]:
-        """Get list of running servers"""
-        host_list = await self.client.get_text("/cluster/running-servers")
-        return host_list.split(",")
+    async def running_servers(self) -> List[ServerInfo]:
+        """Get dict of host id to IP address of running servers"""
```

Comment:
Please do a self-review and fix all comments, like this one, to use the proper concepts from the latest version of this PR.
For example here: it is neither a dict, nor does it use Host IDs.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/eae1194fa63f0f18ba03461ec79586c91741e4fc